### ISSUE TITLE
The documentation example for visualize_image_attr_multiple seems puzzling 

### ISSUE THREAD

**Issue by mikeaalv** [pytorch/captum#437]
It seems to me the input in this example for visualize_image_attr_multiple is not the same as the argument format for the function `captum.attr.visualization.visualize_image_attr_multiple(attr, original_image, methods, signs, titles=None, fig_size=8, 6, use_pyplot=True, **kwargs)`. It seems to me the part of `methods` and `signs` are in front in this example. I haven't test whether the example can run as it need modification (input data) to run. 

```
>>> # ImageClassifier takes a single input tensor of images Nx3x32x32,
>>> # and returns an Nx10 tensor of class probabilities.
>>> net = ImageClassifier()
>>> ig = IntegratedGradients(net)
>>> # Computes integrated gradients for class 3 for a given image .
>>> attribution, delta = ig.attribute(orig_image, target=3)
>>> # Displays original image and heat map visualization of
>>> # computed attributions side by side.
>>> _ = visualize_mutliple_image_attr(["original_image", "heat_map"],
>>>                     ["all", "positive"], attribution, orig_image)
```

**Comment by vivekmig** [comment]
Hi @mikeaalv , thanks for catching this! You are right, the input ordering is incorrect in this example, attribution and orig_image should be provided first. Would you be able to create a PR fixing this? Thanks!

**Comment by mikeaalv** [comment]
@vivekmig  Definitely glad to do this.

**Comment by vivekmig** [comment]
Resolved in #438 , thanks for fixing this issue @mikeaalv !
